examples/main/main.cpp

```diff
@@ -94,7 +97,7 @@ struct whisper_params {
     grammar_parser::parse_state grammar_parsed;
 };
 
-void whisper_print_usage(int argc, char ** argv, const whisper_params & params);
+void whisper_print_usage(int argc, const char ** argv, const whisper_params & params);
 
 char* whisper_param_turn_lowercase(char* in){
```

Comment:
I think, this needs a `const`.

```suggestion
char* whisper_param_turn_lowercase(const char* in){
```

Reply:
I'm still fixing the compatibility issue. A lot of things need to be tested since we added DTW, and some aspects may not be compatible. Give me a few hours.